sweep/sweeper.go

```diff
@@ -1622,6 +1622,9 @@ func (s *UtxoSweeper) monitorFeeBumpResult(resultChan <-chan *BumpResult) {
 					"fee bump monitor", r.Event,
 					r.Tx.TxHash())
 
+				// Cancel the rebroadcasting of the failed tx.
+				s.cfg.Wallet.CancelRebroadcast(r.Tx.TxHash())
```

Comment:
👍 

I think in a future pass, we could further consolidate the internal rebroadcaster with this fee bumper. At a pass, that would require enabling the sweeper to take a fully signed/complete transaction, unifying all our broadcast paths across the codebase.